### ISSUE TITLE
Camera issue outside of the lan

### ISSUE THREAD

**Issue by n-lefebvre** [galadril/DomoticzForAndroid#189]
As I say to Roady by email, you will find the correct url to call camera snapshot when you are outside of the lan.

So the good template to use is :

http://
external public IP (user gives it in app parameters)
:
NAT port (user gives it also in app parameters)
/
camsnapshot.jpg?idx=
idx number of the camera (get by JSON call)

==> Exemple : 
Camera_Salon ---- http://88.xx.xx.122:68953/camsnapshot.jpg?idx=1
TEST              ---- http://88.xx.xx.122:68953/camsnapshot.jpg?idx=2
TEST 2           ---- http://88.xx.xx.122:68953/camsnapshot.jpg?idx=3

In conclusion the template is the same for all Cameras from outside just change the idx=X at the end, domoticz server does the translatation to get the snapshot in the internal lan ;)

But the main problem is to keep the authentication (Basic-authentifaction / Cookies session) for the snapshot connection otherwise your app show a white picture.

At your disposal if you need anything else ;)
